base_layer/core/src/base_node/comms_interface/inbound_interface.rs

```diff
@@ -44,9 +46,28 @@ where T: BlockchainBackend
     /// Handle inbound node comms requests from remote nodes.
     pub async fn handle_request(&self, request: &NodeCommsRequest) -> Result<NodeCommsResponse, CommsInterfaceError> {
         match request {
+            // TODO: replace with async calls
```

Comment:
This is easy to do with the `async_db` module

Reply:
Cool, I will implement that change in the next PR.